src/components/Modals/CreateGnosisSafeModal/index.tsx

```diff
@@ -87,6 +97,19 @@ const CreateGnosisSafeModalContent: FunctionComponent<{
 				members.reduce((acc, cur) => acc && !!cur, true)
 			))
 
+	if (created) {
+		return (
+			<div className="create-gnosis-safe">
+				<h2>{created.includes("success") ? "Success!" : "Failed!"}</h2>
+
+				<div className="create-gnosis-safe__row">
+					You can now access your DAO dashboard / information on the &ldquo;View Your DAOs &rdquo;
```

Comment:
Even if `created` does not include "success" - you will be displaying a message that the user can access his DAO dashboard. Quite confusing.